scripts/dataset_processing/tts/ljspeech/get_data.py

```diff
@@ -56,7 +56,7 @@ def __extract_file(filepath, data_dir):
 def __process_data(data_root, whitelist_path):
     if whitelist_path is None:
         wget.download(
-            "https://raw.githubusercontent.com/NVIDIA/NeMo/main/nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv",
+            "https://raw.githubusercontent.com/NVIDIA/NeMo/main/scripts/tts_dataset_files/en/whitelist_lj_speech.tsv",
```

Comment:
this would mess up the versioning of the file tied to the versioning of dependency `nemo_text_processing`. In your requirement, `nemo_text_processing==0.1.6rc0`, but `main` is used here.

Reply:
> this would mess up the versioning of the file tied to the versioning of dependency `nemo_text_processing`. In your requirement, `nemo_text_processing==0.1.6rc0`, but `main` is used here.

This shouldn't be an issue as `lj_speech.tsv` is based on the file that was released with the dataset, it's not going to be updated.

Reply:
yes. just double-checked the whitelist was described in its README.